.eslintrc.js

```diff
@@ -43,7 +43,7 @@ module.exports = {
     "quote-props": ["error", "as-needed"],
     "no-prototype-builtins": "error",
     "no-array-constructor": "error",
-    "prefer-destructuring": "warn",
+    "prefer-destructuring": "off",
```

Comment:
Is er een reden om dit in deze PR te doen en niet rechtstreeks op development ?

Reply:
Het is hier voorgekomen kon misschien beter op development gedaan worden.. op zich maakt het hier geen functioneel verschil dus zouden we deze gewoon kunnen laten doorstromen... je gaat hier niks van merken.